contributing_to_docs/doc_guidelines.adoc

```diff
@@ -61,7 +61,7 @@ topic titled "Managing Authorization Policies".
 
 == Topic Titles and Section Headings
 
-Use sentence case in all topic titles and section headings. See http://www.titlecase.com/ or
+Use title case in all topic titles and section headings. Note that this is different from {product-title} 4.x docs that use sentence case. See http://www.titlecase.com/ or
```

Comment:
`{product-title}` will only render in AsciiBinder builds, so can just type out the product name here.

Reply:
fixed